networkit/_NetworKit.pyx

```diff
@@ -1258,6 +1258,9 @@ cdef class SSSP(Algorithm):
 		
 		Returns a vector of nodes ordered in increasing distance from the source.
 
+		For this functionality to be available, storeNodesSortedByDistance has to be set to true in the constructor.
+		There are no guarantees regarding the order of two nodes that have the same distance to the source.
```

Comment:
Small edits: replace "order" with "relative ordering" and "that have" with "with". Here and elsewhere.